cmd/ttn-lw-stack/commands/config.go

```diff
@@ -48,6 +51,7 @@ type Config struct {
 	GCS              gatewayconfigurationserver.Config `name:"gcs"`
 	DTC              devicetemplateconverter.Config    `name:"dtc"`
 	QRG              qrcodegenerator.Config            `name:"qrg"`
+	PBA              packetbrokeragent.Config          `name:"pba"`
```

Comment:
Can we also add a page to the configuration reference in the documentation?

Reply:
Yes, see original comment, that will be part of this PR